app/src/main/AndroidManifest.xml

```diff
@@ -30,7 +31,8 @@
             android:configChanges="orientation|screenSize"
             android:label="@string/app_name"
             android:name="org.mifos.mobile.ui.activities.LoginActivity"
-            android:windowSoftInputMode="adjustResize"/>
+            android:windowSoftInputMode="adjustResize"
+            android:theme="@style/LoginBackground"/>
```

Comment:
- instead of adding a theme here, can you add this image as background at the activity level. Can you check on the feasibility of it?
- another thing, which I observed, can we use `vector resource` instead of `png`?

Reply:
adding the image as background at the activity level compresses it when the keyboard is opened.  Besides, I have only added this theme in the corresponding activity tag in the `AndroidManifest.xml` file.

I didn't have the `vector resource` of the image. So, I had to use the `png` file.